anchore-policy-validator/templates/validator-rbac.yaml

```diff
@@ -70,3 +70,16 @@ roleRef:
 subjects:
 - kind: ServiceAccount
   name: {{ template "anchore-policy-validator.fullname" . }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: unauth-discovery
```

Comment:
As discussed on Slack, let's add some anchore specific name to avoid future incompatibility.